distsql/request_builder_test.go

```diff
@@ -285,7 +285,7 @@ func (s *testSuite) TestRequestBuilder1(c *C) {
 	expect := &kv.Request{
 		Tp:      103,
 		StartTs: 0x0,
-		Data:    []uint8{0x8, 0x0, 0x18, 0x0, 0x20, 0x0, 0x40, 0x0, 0x5a, 0x0},
```

Comment:
We may want to keep the test passing in both big-endian machines and little-endian machines.

Reply:
The `Data` []uint8 is set by dag.Marshal(), and it can guarantee same in all machines by google/protobuf if the DAGRequest is same.